apps/src/templates/studioHomepages/CourseBlocksGradeBands.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import ContentContainer from '../ContentContainer';
+import ResourceCard from './ResourceCard';
+import styleConstants from '../../styleConstants';
+import i18n from "@cdo/locale";
+
+const pegasusContentWidth = styleConstants['content-width'];
+
+const styles = {
+  container: {
+    width: pegasusContentWidth,
+    display: "flex",
+    justifyContent: "space-between"
+  }
+};
+
+const CourseBlocksGradeBands = React.createClass({
```

Comment:
`React.createClass` is deprecated and won't be supported once we update (in the next couple of weeks, I hope).  Prefer ES6 classes (`class CourseBlocksGradeBands extends React.Component { ...`) or functional stateless components (`function CourseBlocksGradeBands(props) { ...`)